DEFAULT_CONFIG.json5

```diff
@@ -471,14 +471,14 @@
           /// then amount of memory being allocated for each queue is SIZE_XXX * LINK_MTU.
           /// If qos is false, then only the DATA priority will be allocated.
           size: {
-            control: 2,
-            real_time: 2,
-            interactive_high: 2,
-            interactive_low: 2,
+            control: 1,
+            real_time: 1,
+            interactive_high: 1,
+            interactive_low: 1,
             data_high: 2,
-            data: 2,
-            data_low: 2,
-            background: 2,
+            data: 4,
```

Comment:
I've reverted the previous commit, so the numbers are back to the original ones. Throughputs of big messages was indeed very impacted by a smaller batch number, 130kmsg/s -> 50kmsg/s for 70kB messages.
@Mallets Do you think we should modify the numbers? for example increasing the hight priority to 2? In fact this PR does lazy allocation, and only keep one buffer in memory, so we could put any numbers, it would not change memory consumption at all, it just a matter of throughput/congestion.